src/js/components/Video/Video.js

```diff
@@ -210,6 +212,16 @@ const Video = forwardRef(
       [duration, videoRef],
     );
 
+    const seekForward = useCallback(() => {
+      setInteracting(true);
+      videoRef.current.currentTime += skipInterval || 5;
```

Comment:
Is 5 just the standard number? Just want to know how you decided on 5

Reply:
I saw other video players using 5 seconds but I couldn't find any info on if there is a standard for this

Reply:
Netflix and I think YouTube tend to use 10s, at list on their skip forward and backward buttons.

Reply:
Might be nice to have the default specified in the theme, so that callers can set their own defaults as opposed to each instance.

Reply:
If we have the default set in the theme should we still have the `skipInterval` prop to override the default?

Reply:
yes, I would keep the prop -- nice to have ability to override and not lock folks in

Reply:
Sounds good, I will make these changes